<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T_Rax_MainWindow</class>
 <widget class="QMainWindow" name="T_Rax_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>654</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>T-Rax ver 0.203</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget, QTabWidget::tab-bar,  QTabWidget::panel, QWidget{  
     background: rgba(30, 30, 30, 255);      
 }  
   
 QLabel , QCheckBox, QGroupBox, QRadioButton, QComboBox  {  
    background: rgba(0,0,0,0);
     color: #F1F1F1;  
 }  


QTabWidget::tab-bar{ 
    alignment: center;
}

QWidget{
    color: #F1F1F1;
}


QTabBar::tab:left, QTabBar::tab:right {  
     background: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(30, 30, 30, 255), stop:1 #505050);
     border: 1px solid  #5B5B5B;  
    font: normal 14px;
    color: #F1F1F1;
     border-radius:2px;
    
    padding: 0px;
     width: 20px;  
    min-height:140px;
 }  


QTabBar::tab::top, QTabBar::tab::bottom {  
     background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(30, 30, 30, 255), stop:1 #505050);
     border: 1px solid  #5B5B5B;  
    border-right: 0px solid white;
      color: #F1F1F1; 
    font: normal 11px;
     border-radius:2px;
     min-width: 80px;  
    height: 19px;
    padding: 0px;
     margin-top: 1px ;
    margin-right: 1px;
 }  
QTabBar::tab::left:last, QTabBar::tab::right:last{
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}
QTabBar::tab:left:first, QTabBar::tab:right:first{
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
}

QTabWidget, QTabWidget::tab-bar,  QTabWidget::panel, QWidget{  
     background: rgba(30, 30, 30, 255);      
 }  

QTabWidget::tab-bar {
    alignment: center;
}

 QTabBar::tab:hover {  
     border: 1px solid #ADADAD;  
 }  
   
 QTabBar:tab:selected{  

    background: qlineargradient(
        x1: 0, y1: 1, 
        x2: 0, y2: 0,
        stop: 0 #727272, 
        stop: 1 #444444
    );
     border:1px solid  rgb(255, 120,00);/*#ADADAD; */ 
}

QTabBar::tab:bottom:last, QTabBar::tab:top:last{
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}
QTabBar::tab:bottom:first, QTabBar::tab:top:first{
    border-top-left-radius: 10px;
    border-bottom-left-radius: 10px;
}
 QTabBar::tab:top:!selected {  
    margin-top: 1px;
    padding-top:1px;
 }  
QTabBar::tab:bottom:!selected{
    margin-bottom: 1px;
    padding-bottom:1px;
}

QGraphicsView {
    border-style: none;
}

 QLabel , QCheckBox, QGroupBox, QRadioButton, QListWidget::item, QPushButton, QToolBox::tab, QSpinBox, QDoubleSpinBox , QComboBox{  
     color: #F1F1F1; 
    font-size: 12px;
 }  
 QCheckBox{  
     border-radius: 5px;  
 }  
 QRadioButton, QCheckBox {  
     font-weight: normal;  
    height: 15px;
 }  
 
 QLineEdit  {  
     border-radius: 2px;  
     background: #F1F1F1;  
     color: black;  
    height: 18 px;
 }  

QLineEdit::focus{
    border-style: none;
     border-radius: 2px;  
     background: #F1F1F1;  
     color: black;  
}
QSpinBox, QDoubleSpinBox {
    background-color:  #F1F1F1; 
    color: black;
    margin-left: -15px;
    margin-right: -2px;
    height: 30px;
}

QComboBox QAbstractItemView::item { 
	min-height: 40px; 
	min-width: 60px; 
}
QComboBox QAbstractItemView{
    background: rgba(35, 35, 35, 255);
    color: #F1F1F1;
	min-height: 50px;
    selection-background-color: rgba(221, 124, 40, 120);
    border-radius: 5px;

}

QComboBox:!editable {
    margin-left: 1px;
    padding-left: 10px;
    height: 23px;
    background-color: rgba(35, 35, 35, 255);
}

QComboBox::item{
    background-color: rgba(35, 35, 35, 255);
}

QComboBox::item::selected {
    background-color: #505050;
}


QToolBox::tab:QToolButton{
    background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(30, 30, 30, 255), stop:1 #505050);
     border: 1px solid  #5B5B5B;  

     border-radius:2px;
     padding-right: 10px;  
    
      color: #F1F1F1; 
    font-size: 12px;
    padding: 3px;
}
QToolBox::tab:QToolButton{
    background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(30, 30, 30, 255), stop:1 #505050);
     border: 1px solid  #5B5B5B;  

     border-radius:2px;
     padding-right: 10px;  
    
      color: #F1F1F1; 
    font-size: 12px;
    padding: 3px;
}
  
QPushButton{  
     color: #F1F1F1;
     background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #323232, stop:1 #505050);
     border: 1px solid #5B5B5B;
     border-radius: 5px; 
     padding-left: 8px;
height: 18px;
    padding-right: 8px;   
 }  
QPushButton:pressed{
        margin-top: 2,px;
        margin-left: 2px;   
}
QPushButton::disabled{
}

QPushButton::hover{  
     border:1px solid #ADADAD; 
 }  
 

QPushButton::checked{
    background: qlineargradient(
        x1: 0, y1: 1, 
        x2: 0, y2: 0,
        stop: 0 #727272, 
        stop: 1 #444444
    );
     border:1px solid  rgb(255, 120,00);
}

QPushButton::focus {
    outline: None;
}
 QGroupBox {  
     border: 1px solid #ADADAD;  
     border-radius: 4px;  
     /*margin-top: 7px;*/  
     padding: 0px  
 }  
 QGroupBox::title {  
      subcontrol-origin: margin;  
      left: 20px  
  }

QSplitter::handle:hover {
    background: rgba(30, 30, 30, 255);
 }


QGraphicsView{
    border-style: none;
}

 QScrollBar:vertical {
      border: 2px solid rgba(30, 30, 30, 255);
      background: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 #323232, stop:1 #505050);
      width: 12px;
      margin: 0px 0px 0px 0px;
  }
  QScrollBar::handle:vertical {
      background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #969696, stop:1 #CACACA);
     border-radius: 3px;
      min-height: 20px;
    padding: 15px;
  }
  QScrollBar::add-line:vertical {
      border: 0px solid grey;
      height: 0px;
  }

  QScrollBar::sub-line:vertical {
      border: 0px solid grey;
      height: 0px;
  }
  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }



#navigation_frame {
    border: none;
    background: qlineargradient(spread:reflect, x1:0, y1:0.5, x2:0, y2:0, stop:0.114428 rgba(21, 134, 31, 255), stop:0.467662 rgba(12, 80, 18, 255), 
    stop:0.726368 rgba(9, 60, 13, 255), stop:1 rgba(0, 0, 0, 255))
}



#navigation_frame QPushButton{
    min-width: 104px;
    min-height: 20px;
	font-size: 12px;
	padding: 2px;
    font-weight: bold;
    border-radius: 10px;
}

#temperature_btn {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.20398 rgba(221, 124, 40, 180), stop:1 rgba(0, 0, 0, 255))
}
#ruby_btn {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.20398 rgba(197, 0, 3, 255), stop:1 rgba(0, 0, 0, 255))
}

#diamond_btn {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.20398 rgba(27, 0, 134, 255), stop:1 rgba(0, 0, 0, 255))
}
#raman_btn {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.20398 rgba(21, 134, 31, 255), stop:1 rgba(0, 0, 0, 255))
}
</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="navigation_frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="temperature_btn">
         <property name="text">
          <string>Temperature</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ruby_btn">
         <property name="text">
          <string>Ruby</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="diamond_btn">
         <property name="text">
          <string>Diamond</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="raman_btn">
         <property name="text">
          <string>Raman</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>248</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>written by C. Prescher, GSECARS, UofC</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="axes_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="main_frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="figure1_frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="figure2_frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="figure3_frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="status_bar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="status_coord_lbl">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress_bar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>18</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white</string>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>743</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="status_file_information_lbl">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
